src/app/components/EnterPage/enter.scss

```diff
@@ -0,0 +1,73 @@
+@import url('https://fonts.googleapis.com/css?family=Mulish');
+
+:local {
+  .enter {
+    position: fixed;
+    width: 426px;
+    height: 238px;
+    left: calc(747/1920*100%);
+    top: 96px;
+
+    padding: 0px;
+    background: #FFFFFF;
+
+    box-shadow: 0px 0px 12px rgba(188, 188, 188, 0.25);
+
+    a {
+      text-decoration: none;
+      color:#1C83C6;
```

Comment:
If you are using Visual Studio Code or Atom, I recommend a plugin called `prettier` to format your files.

Reply:
Oops, I forgot to add stylelint for our project

Reply:
I forgot to add stylelint for our project